dapr/clients/grpc/client.py

```diff
@@ -289,7 +289,8 @@ def publish_event(
             pubsub_name: str,
             topic_name: str,
             data: Union[bytes, str],
-            metadata: Optional[MetadataTuple] = ()) -> DaprResponse:
+            metadata: Optional[MetadataTuple] = (),
+            data_content_type: Optional[str] = None) -> DaprResponse:
```

Comment:
Default to application/json instead of None.

Reply:
Why? We can take a blob here, and this is an optional field in the cloud event spec?

Reply:
OK. Let the runtime decide on the default behavior.